Resources/Prototypes/Roles/Jobs/Command/blue_shield.yml

```diff
@@ -0,0 +1,43 @@
+- type: job
+  id: Blueshield

```

Comment:
Personally, I'm not a fan of blueshield being a job. IMO it should be all the gear in a locker and the captain can appoint someone.

Reply:
I'll add a mailbox later. Maybe in the evening.

Reply:
I made a closet